### ISSUE TITLE
[ZOOM]: Disability Calculator - Review layout for visual UI at or above 300%

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#423]
## Issue
The input fields start getting hard to read when I set my browser width to 1280px and zoom in to 300% or 400%. We are in general compliance for the time being with the layout functioning well at 200%, but WCAG 2.1 asks for a 400% zoom compliance, so I'm being proactive. Screenshot attached below.

## Acceptance Criteria
As a user who likes to zoom my layout, I would like all inputs to be readable at 400%

## Environment
* MacOS Mojave
* Chrome latest
* 300 and 400% zoom at 1280px width

## WCAG or Vendor Guidance (optional)
* [Understanding Reflow](https://w3c.github.io/wcag21/understanding/21/reflow.html)

## Screenshots or Trace Logs
![Screen Shot 2019-06-27 at 12 58 26 PM](https://user-images.githubusercontent.com/934879/60289585-6074b880-98dc-11e9-9c80-5198a85c20cb.png)

**Comment by jenniferlee-dsva** [comment]
Thank you @1Copenut. I've @rtwell here to address the column headers issue ^.  Are you also saying that the number input field has a problem? Didn't quite understand that one.

**Comment by 1Copenut** [comment]
@jenniferlee-dsva Yes, the number field becomes hard to read at 400%, and the labels start wrapping to two lines. This screenshot shows the labels becoming close together at 300%, but if you continue zooming, the layout becomes more condensed.

**Comment by ncksllvn** [comment]
Hey @rtwell, I think this is the ticket with the potential design impact for the Disability Calc. Do you want to explore possible solutions for this while I tackle other 508 issues? Sounds like it may be as simple as expanding the width of the rating inputs but obviously could lead to a little more.

**Comment by ncksllvn** [comment]
Adding the `design` label to indicate this isn't `frontend`-ready yet.

**Comment by rtwell** [comment]
hey gang—here is what the calc should look like at +300% zoom (coincidently, it's also what it should look like on x-small screens!). cc @1Copenut @ncksllvn 

Please note, the delete "button" hides the word `delete` and _displays_ only the trash can icon, however, a screenreader would be identical to desktop.

![image](https://user-images.githubusercontent.com/12564977/63437008-c0c53980-c3f7-11e9-9f04-b675e542a324.png)

**Comment by ncksllvn** [comment]
Here it is currently at 300%. I see the label changed to `Description (optional)` (which I assume applies to all screen sizes/zoom levels), but otherwise, is there anything left here that I'm missing?

![image](https://user-images.githubusercontent.com/1915775/63950899-d6b3a980-ca4a-11e9-82b0-4c5dfff533cd.png)
![image](https://user-images.githubusercontent.com/1915775/63950914-e03d1180-ca4a-11e9-9a66-903a3b520266.png)

**Comment by ncksllvn** [comment]
Ah, and it looks like the buttons need to be centered and full-width.

**Comment by jenniferlee-dsva** [comment]
This is connected to: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18214 and this https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/16956
[just adding central issue and related legacy rewriting tickets - i'm having trouble finding all the related dependency things]

**Comment by Karak888** [comment]
Not sure of status here so I am going to leave it for now @jenniferlee-dsva

**Comment by Karak888** [comment]
@jenniferlee-dsva I am closing this one out, but you will still be able to access the connected issues as they are captured in the ticket you referenced above (which is still in vets.gov repo).
